java/vector/src/main/java/org/apache/arrow/vector/compare/RangeEqualsVisitor.java

```diff
@@ -225,7 +228,7 @@ protected boolean compareStructVectors(Range range) {
     }
 
     for (String name : leftChildNames) {
-      RangeEqualsVisitor visitor = createInnerVisitor(leftVector.getChild(name), rightVector.getChild(name));
+      RangeEqualsVisitor visitor = createInnerVisitor(leftVector.getChild(name), rightVector.getChild(name), null);
```

Comment:
please add a comment for what null represents (/*param_name=*/) here and below.

Reply:
Comments added. Thanks for the good suggestion.